.travis.yml

```diff
@@ -1,7 +1,18 @@
 language: ruby
 rvm:
-- 2.1.0
 - 2.2.2
+- 2.4.0
+- 2.4.1
+bundler_args: --without development
+gemfile:
+  - gemfiles/rails_5.0.gemfile
+  - gemfiles/rails_5.1.gemfile
+matrix:
+  allow_failures:
+    - rvm: 2.4.0
+    - rvm: 2.4.1
```

Comment:
Why are there failures on 2.4.0 and 2.4.1? Seems slightly concerning.

Reply:
@pencil something is causing the `appraisal` gem to not load properly on Travis to be able to run the tests - started looking into it, but not conclusive yet